python-backend/app/api/mines/status/models/mine_status_xref.py

```diff
@@ -48,6 +48,10 @@ def json(self):
             if self.mine_operation_status_sub_reason_code else {},
         }
 
+    @classmethod
+    def active_status_codes(cls):
```

Comment:
Small naming suggestion. Since we already know we're dealing with status codes (based on the class), this can probably just be `active` or `active_records`. It would be nice to have a consistent method name across all/most of our classes for accessing active records, imo

Reply:
Changed to just be active.